pre_publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+cd clippy_lints && cargo fmt && cd ..
+cargo fmt
```

Comment:
Can it add `--write-mode=overwrite` if the repo is clean? I hate those .bk files